### ISSUE TITLE
Improve accessibility by adding aria label tags to buttons on the Documents page

### ISSUE THREAD

**Issue by pkaufhol** [CMU-313/Teedy#325]
The buttons on the Documents page should have aria labels, so users with screen readers are able to get a sense of what the button does instead of it just saying "button". Making this change will also improve the Lighthouse accessibility score.

**Comment by pkaufhol** [comment]
I would like to work on this please!
